airflow/www/app.py

```diff
@@ -65,6 +66,7 @@ def create_app(config=None, session=None, testing=False, app_name="Airflow"):
     app.config['SESSION_COOKIE_HTTPONLY'] = True
     app.config['SESSION_COOKIE_SECURE'] = conf.getboolean('webserver', 'COOKIE_SECURE')
     app.config['SESSION_COOKIE_SAMESITE'] = conf.get('webserver', 'COOKIE_SAMESITE')
+    app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=3560)  # 10 years
```

Comment:
This should either come from a config, or be set before line 61 (where we do `app.config.from_pyfile`) so that this setting can be changed from webserver_config.py.

I think a 10year default is a bit too long a default too :) 14 or 30 days seems more reasonable

Reply:
Exposed as a configuration.